src/didactWebView.ts

```diff
@@ -93,7 +94,11 @@ export class DidactWebviewPanel {
 
 	public static createOrShow(extensionPath: string, inpath?: vscode.Uri | undefined, column?: ViewColumn) {
 		if (!column) {
-			column = vscode.window.activeTextEditor ? vscode.window.activeTextEditor.viewColumn : ViewColumn.One;
+			// if we weren't passed a column, use the last column setting
+			column = getLastColumnUsedSetting();
+		} else {
+			// if we are passed a column, stash it
+			setLastColumnUsedSetting(column);
```

Comment:
Can it happen that the column number is invalid? (a number too high and the column doesn't exist?)